core/ordering/cosipbft/blockstore/mem.go

```diff
@@ -180,7 +185,11 @@ type observer struct {
 }
 
 func newObserver(ctx context.Context, watcher core.Observable) *observer {
+	// This channel must have a buffer of size 1, no more no less, in order to
+	// preserve the ordering of the events but also to prevent the observer
+	// buffer to be used when the channel is correctly listened to.
```

Comment:
that kind of comment is very precious, thanks ! 👍

Reply:
Agreed 👍